CyclopsNuclearReactor/Helpers/DirectoryHelper.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace CyclopsNuclearReactor.Helpers
+{
+    using System;
+    using System.IO;
+
+    public static class DirectoryHelper
+    {
+        public static string GrabFromAssetsDirectory(string modName, string file)
```

Comment:
I don't see this being used anywhere, but if this were to be ever used to get the mod's own assets, I would recommend getting the path from Assembly.GetExecutingAsswmbly().Location instead of using hardcoded folder names

Reply:
This was used in this very same pull request to register a sprite that isn't attached to `TechType`